src/promptflow-tools/promptflow/tools/common.py

```diff
@@ -656,30 +709,40 @@ def unescape_roles(val, escape_dict: dict):
         return val
 
 
+def escape_roles_for_flow_inputs_and_prompt_output(escape_dict: dict, inputs_to_escape: list, **kwargs):
```

Comment:
The method name is weird, becuase the method implementation is not related to flow inputs or prompt output.